### PR TITLE
Halfspace representation of an empty VPolytope

### PR DESCRIPTION
**Pull request by mforets**
Fixes the corner case `tohrep(VPolytope())` reported [here](https://github.com/JuliaReach/LazySets.jl/issues/947).

Note: if this gets merged, for consistency we should use the same convention for `VPolygon`,  which gives a polygon with no constraints in `#master`:

```julia
julia> tohrep(VPolygon())
HPolygon{Float64}(HalfSpace{Float64}[])
```

### CODE REVIEW

**Comment by schillic** [comment]
> if this gets merged, for consistency we should use the same convention for VPolygon

Let us do that here as well to avoid further confusion.

File: src/VPolytope.jl [hunk 1]
```diff
@@ -426,6 +426,11 @@ For further information on the supported backends see
 """
 function tohrep(P::VPolytope{N};
                 backend=default_polyhedra_backend(P, N)) where {N<:Real}
+    vl = P.vertices
+    n = length(vl)
+    if n == 0
```
Comment (schillic) [inline src/VPolytope.jl]: Shorter: `if isempty(vl)`

File: test/unit_Polygon.jl [hunk 2]
```diff
@@ -396,4 +400,4 @@ end
 # default Float64 constructors
 @test HPolygon() isa HPolygon{Float64}
 @test HPolygonOpt() isa HPolygonOpt{Float64}
-@test VPolygon() isa VPolygon{Float64}
+@test VPolygon() isa VPolygon{Float64}
```
Comment (schillic) [inline test/unit_Polygon.jl]: Why remove the empy line here?

Comment (mforets) [reply test/unit_Polygon.jl]: (it was unintentional)

Thanks for the review. I'll update those changes and merge when CI passes.

**Pull request merged**